scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -40,7 +40,25 @@ RUN set -ex; \
 		libcvc4-dev libz3-static-dev z3-static jq \
 		; \
 	apt-get install -qy python3-pip python3-sphinx; \
-	pip3 install codecov; \
+	pip3 install codecov;
+
+# Eldarica
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -qy unzip default-jre; \
+	download_url=$(curl \
+		--silent \
+		--location \
+		--fail \
+		--show-error \
+		https://api.github.com/repos/uuverifiers/eldarica/releases/latest \
```

Comment:
Is it preferred to get the latest release?

Wouldn't it be a problem if something happens in a future Eldarica release that will break our tests?
Or is the reason so that these scripts would not have to be updated for every new release?

Reply:
Since these images are not updated frequently, any potential issues with Eldarica releases would likely occur only when we rebuild those docker images. Originally, my intention was indeed to avoid the need for script updates. However, now that we're adding support to OSX as well and the OSX dependency scripts run more frequently than we update the Docker images, it's indeed better to fix the Eldarica version to prevent any potential version mismatches.